cmd/podman/shared/container.go

```diff
@@ -665,7 +673,7 @@ func GenerateRunlabelCommand(runLabel, imageName, name string, opts map[string]s
 		return ""
 	}
 	newS := os.Expand(strings.Join(cmd, " "), envmapper)
-	cmd = strings.Split(newS, " ")
+	cmd = strings.Fields(newS)
```

Comment:
I think both are dangerous as both will cause issues for strings (e.g., `--option="some string with spaces"`). I think we should use `shlex.Split(..)` as we do in `GenerateCommand`.

Reply:
Done.